src/pages/Login/components/LoginForm.tsx

```diff
@@ -52,13 +52,14 @@ export const LoginForm = (props: LoginFormProps) => {
 			setSuccessMessage(
 				"You have been logged in successfully. You will be redirected shortly.",
 			);
+			console.log(login);
 		} catch (error) {
 			// Get error from api
 			const loginError =
 				((error as ApiError).response?.data?.error as String) ||
 				"Unknown error";
 			setSignedIn(false);
-			console.log(`There was an error while logging in: ${error}`);
+			console.log(`There was an error while logging in: ${loginError}`);
```

Comment:
probably better to use console.error here

Reply:
You're right. Though it would leave unused loginError, and our setup does not deploy with unused locals

Reply:
Oh I mean use console.error, not console.log. Keep usage of the same variables.

Reply:
Ah, I misunderstood. You're correct